applications/registration-api/src/main/java/no/dcat/service/ElasticsearchService.java

```diff
@@ -31,6 +31,12 @@
             elasticClient.registerSetting("reg-api-catalog", mapper.readTree(new ClassPathResource("reg-api-catalog.settings.json").getInputStream()).toString());
             elasticClient.registerMapping("reg-api-catalog", "apicatalog", mapper.readTree(new ClassPathResource("apicatalog.mapping.json").getInputStream()).get("apicatalog").toString());
             elasticClient.initializeAliasAndIndexMapping("reg-api-catalog");
+
+            elasticClient.registerSetting("register", mapper.readTree(new ClassPathResource("register.settings.json").getInputStream()).toString());
+            elasticClient.registerMapping("register", "dataset", mapper.readTree(new ClassPathResource("register.dataset.mapping.json").getInputStream()).get("dataset").toString());
+            elasticClient.registerMapping("register", "catalog", mapper.readTree(new ClassPathResource("register.catalog.mapping.json").getInputStream()).get("catalog").toString());
```

Comment:
Mulig jeg misforstår noe her, men kan man ikke bare ha en type pr index? Her ser det ut som du prøver å lagge mapping for 2 (dataset og catalog).

Reply:
register-api var lagt før vi (eller elasticsearch) bestemt. Det er tillatt, men dårlig ide å ha flere typer i en index.
i elasticsearch 7 kommer det bli forbudt, men vi er i 5nå